src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2972,6 +2972,21 @@
     /**
      * The {@code >} (greater than) operator.
      *
+     * <p>For example:</p>
+     * <pre>
+     * c=a &gt; b;
```

Comment:
please respect spacing in examples
should be `c = a &gt; b;`

Reply:
Okay got it

Reply:
Yep Done! Added a space after <code>=</code> in <code>c=a &gt; b;</code>